src/packages/lib.ts

```diff
@@ -18,13 +18,14 @@ export { default as GridToggler } from '../h5web/toolbar/controls/GridToggler';
 
 // Building blocks
 export { default as VisCanvas } from '../h5web/vis-packs/core/shared/VisCanvas';
+export { default as AxisSystemProvider } from '../h5web/vis-packs/core/shared/AxisSystemProvider';
+export { default as AxisSystem } from '../h5web/vis-packs/core/shared/AxisSystem';
```

Comment:
Is there still an interest to export these if everything is inside `VisCanvas` ?

Reply:
🎣